src/Microsoft.AspNetCore.Authentication/RemoteAuthenticationOptions.cs

```diff
@@ -23,11 +23,6 @@ public override void Validate()
             {
                 throw new ArgumentException(Resources.FormatException_OptionMustBeProvided(nameof(CallbackPath)), nameof(CallbackPath));
             }
-
-            if (string.IsNullOrEmpty(SignInScheme))
-            {
-                throw new ArgumentException(Resources.FormatException_OptionMustBeProvided(nameof(SignInScheme)), nameof(SignInScheme));
```

Comment:
Man updating this is logigng stuff terrible, is really the logging pattern we are supposed to use everywhere?